README.md

```diff
@@ -30,8 +30,8 @@ Notify a slack channel with a custom message at any point in a job with this cus
 
 | Parameter | Type | Default | Description |
 |-----------|------|---------|-------------|
-| `webhook` | `string` | ${SLACK_WEBHOOK} | Either enter your webhook value or use the CircleCI UI to add your token under the `SLACK_WEBHOOK` environment variable |
-| `message` | `string` | Your job on CircleCI has completed. | Enter a custom message to send to your Slack channel |
+| `webhook` | `env_var_name` | ${SLACK_WEBHOOK} | Please use the CircleCI UI to add your token under the `SLACK_WEBHOOK` environment variable |
```

Comment:
I think maybe we should mention that they can name the env var anything they want, then it will be more clear how to set multiple webhooks. Perhaps we could have an example such as `SLACK_WEBHOOK_1`

Reply:
Here as well would not have ${}